framework/src/base/MooseApp.C

```diff
@@ -307,6 +307,14 @@ MooseApp::getPrintableVersion() const
 void
 MooseApp::setupOptions()
 {
+  // MOOSE was updated to have the ability to register execution flags in similar fashion as
+  // objects. However, this change requires all *App.C/h files to be updated with the new
+  // registerExecFlags method. To avoid breaking all applications the default MOOSE flags
+  // are added if nothing has been added to this point. In the future this could go away or
+  // perhaps be a warning.
+  if (_execute_flags.items().empty())
+    Moose::registerExecFlags(_factory);
```

Comment:
Does this also mean that every derived app has to call `Moose::registerExecFlags()`??

Reply:
No, this is what allows apps to work without calling Moose::registerExecFlags() directly. This is in setupOptions within MooseApp and should be called for all derived apps automatically.

Reply:
Ok